packages/cli/tests/create-tutorial.test.ts

```diff
@@ -67,6 +68,20 @@ test('create and build a project', async (context) => {
   const distFiles = await fs.readdir(path.join(dest, 'dist'), { recursive: true });
 
   expect(distFiles.map(normaliseSlash).sort()).toMatchSnapshot();
+
+  // create snapshot of lesson, solution and template file reference JSONs
+  const lessonJsons = distFiles.filter((file) => file.endsWith('-files.json'));
+  const solutionJsons = distFiles.filter((file) => file.endsWith('-solution.json'));
+  const templateJsons = distFiles.filter((file) => file.startsWith('template-') && file.endsWith('.json'));
+
+  const contents = [...lessonJsons, ...solutionJsons, ...templateJsons].reduce((jsons, current) => {
+    const fileJson = JSON.parse(readFileSync(path.join(dest, 'dist', current), 'utf8'));
+    const filenames = Object.keys(fileJson);
+
+    return { ...jsons, [current]: filenames };
+  }, {});
+
+  expect(JSON.stringify(contents, null, 2)).toMatchSnapshot('built project file references');
```

Comment:
This assertion is unrelated to the bug. I first thought the problem would be here and created this test case to see if it passes on Windows - it did. 

It seems useful so let's include it as well.